scripts/watchdog.ps1

```
@@ -0,0 +1,316 @@
+        }
+        $State.lastAction = "server_start_failed"
+        $State.lastError = "Unable to start server process ($Reason)."
+        return $false
+    } catch {
+        $State.lastAction = "server_start_failed"
+        $State.lastError = $_.Exception.Message
+        return $false
+    }
+}
+
+function Ensure-EdgeRunning {
```

PSScriptAnalyzer (1.24.0)

[warning] The cmdlet 'Ensure-EdgeRunning' uses an unapproved verb.

(PSUseApprovedVerbs)